packages/components/src/combobox-control/stories/index.js

```diff
@@ -257,6 +262,9 @@ const countries = [
 export default {
 	title: 'Components/ComboboxControl',
 	component: ComboboxControl,
+	parameters: {
+		knobs: { disable: false },
+	},
```

Comment:
The `knobs` addon is deprecated and we're in the process of converting the examples to the new `controls` approach.

Would you mind refactoring this Storybook example? Here's a couple of references:

- [docs](https://storybook.js.org/docs/react/essentials/controls)
- [a component in gutenberg using controls in its storybook example](https://github.com/WordPress/gutenberg/blob/9faa63ec01c7db4e4d57823af2db5c54bb6d6660/packages/components/src/toggle-group-control/stories/index.js)

Reply:
Thanks for the links, will take a look later this week.